libraries/rush-lib/src/utilities/npmrcUtilities.ts

```diff
@@ -110,3 +135,9 @@ export function syncNpmrc(
     throw new Error(`Error syncing .npmrc file: ${e}`);
   }
 }
+
+export function isVariableSetInNpmrcFile(sourceNpmrcFolder: string, variableKey: string): boolean {
+  const sourceNpmrcPath: string = path.join(sourceNpmrcFolder, '.npmrc');
+  const trimNpmrcFile: string = _trimNpmrcFile(sourceNpmrcPath);
+  return trimNpmrcFile.includes(`${variableKey}=`);
```

Comment:
This needs to verify that a line starts with this expression, otherwise this will match if `variableKey` is a suffix of some other variable name, or is in a comment, etc.

Reply:
Thanks! Changed to use RegExp to find the matches.